rfcs/0010-ui5-builder-bundling-refactoring.md

```diff
@@ -158,8 +158,6 @@ To be discussed.
 ## Unresolved Questions and Bikeshedding
 *This section should be removed (i.e. resolved) before merging*
 
-Should we generate source maps during bundling if none is provided for a resource? This could be a simple mapping of only the first column of the first line to the original file.
-
 ### resourceRoot-mappings
 
 When bundling modules via resourceRoot-mappings, it might become impossible to reference the original source from a bundle's source map. For example, for a bundle located in `/test-resources`, which includes modules from `/resources`, the source map must not reference the original module via `../../resources/x/y/z` as this path rather depends on the resourceRoot-mapping and might be different at runtime.
```

Comment:
LGTM. 
Line 172 IMO should be moved into the "Source Map support" section, too.